Tests/FoundationEssentialsTests/AttributedString/AttributedStringTests.swift

```diff
@@ -2599,4 +2599,26 @@ E {
         XCTAssertEqual(testConstrainedContainer.filter(inheritedByAddedText: true), AttributeContainer.testInt(2).testParagraphConstrained(3).testCharacterConstrained(4))
         XCTAssertEqual(testConstrainedContainer.filter(inheritedByAddedText: false), AttributeContainer.testNonExtended(5))
     }
+
+    func testWritingDirectionBehavior() {
+        // Indicate that this sentence is primarily right to left, because the English term "Swift" is embedded into an Arabic sentence.
+        var string = AttributedString("Swift مذهل!", attributes: .init().writingDirection(.rightToLeft))
+
+        // To remove the information about the writing direction, set it to `nil`:
+        string.writingDirection = nil
```

Comment:
Is this intentional / did you mean to have some form of assertion between when you set the attribute above and when you removed it here?

Reply:
Forgot the assert 👍